src/metabase/api/embed.clj

```diff
@@ -27,11 +27,14 @@
             [metabase.models
              [card :refer [Card]]
              [dashboard :refer [Dashboard]]
-             [dashboard-card :refer [DashboardCard]]]
+             [dashboard-card :refer [DashboardCard]]
+             [dashboard-card-series :refer [DashboardCardSeries]]]
             [metabase.util :as u]
             [metabase.util
              [embed :as eu]
              [schema :as su]]
+             [metabase
+              [db :as mdb]]
```

Comment:
this indentation is off

Reply:
also we try to keep all our requirements ordered, as you can see. So this should go further up in the file. It should just be `[metabase.db :as mdb`]